modules/seqexec/server/src/main/scala/seqexec/server/SeqexecFailure.scala

```diff
@@ -59,6 +64,20 @@ object SeqexecFailure {
     case GdsException(ex, url)        =>
       s"Failure connecting with GDS at $url: ${ex.getMessage}"
     case GdsXmlError(msg, url)        => s"XML RPC error with GDS at $url: $msg"
+    case FailedSimulation             => s"Failed to simulate"
+  }
+
+  implicit def ev(implicit ae: ApplicativeError[IO, Throwable]): ApplicativeError[IO, SeqexecFailure] = new ApplicativeError[IO, SeqexecFailure] {
```

Comment:
Can we call this `applicativeErrorIOSeqexecFailure` or something like that? It's usually a good idea for implicit definitions to have long and specific names to lessen the risk of shadowing.

Reply:
Yes, I'll do that